### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by nathlazar** [nouzun/udpholepuching#1]
Does it work? i opened a port for the relay server , run two clients behind NAT , relay server detects their presence but the two clients doesn't connect each other

**Comment by tigercoxgithub** [comment]
> Does it work? i opened a port for the relay server , run two clients behind NAT , relay server detects their presence but the two clients doesn't connect each other

Did you get it working?

**Comment by nathlazar** [comment]
yes, I use the local port (which provided by DatagramSocket upon the communication with the relay) to bind it with the new DatagramSocket for communication with another client
